plugins/com.google.cloud.tools.eclipse.appengine.validation.test/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-Vendor: Google Inc.
 Bundle-Version: 0.1.0.qualifier
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.validation
 Require-Bundle: org.junit,
- org.eclipse.equinox.registry
+ org.eclipse.equinox.registry,
+ com.google.cloud.tools.eclipse.test.dependencies
```

Comment:
AFAIK, if we add this bundle as `Required-Bundle`, we can remove all the test-related packages it imports from `Import-Package`. That is, you can remove `org.mockito;provider=google` below.

This has never been streamlined across the entire test bundles, so we might want to do so in the future.